mycroft/configuration/config.py

```diff
@@ -198,6 +199,15 @@ def _log_old_location_deprecation(old_user_config=OLD_USER_CONFIG):
                                              BASE_FOLDER))
 
 
+def _get_system_constraints():
+    # constraints must come from SYSTEM config
+    # if not defined then load the DEFAULT constraints
+    # these settings can not be set anywhere else!
+    return LocalConf(SYSTEM_CONFIG).get("system") or \
+           LocalConf(DEFAULT_CONFIG).get("system") or \
+           {}
```

Comment:
[link to key (below)](https://github.com/OpenVoiceOS/ovos-core/pull/19/files#diff-710a53959602f8d38706b48cad5f274c7da04da59f41079d7b84644c7b3535f6R224-R232) for other reviewers' reference